client/forgotPassword/forgotPassword.ctrl.js

```diff
@@ -22,4 +22,20 @@ module.exports = class ForgotPassword {
       .then(() => this.$scope.$apply())
       .catch(this.errorHandler);
   }
+  sendUsernameRequest () {
+    return Promise.resolve(this.$http({
+      method: 'POST',
+      url: `/api/v1/user/${this.resetEmail}/forgotUsername`
+    })).then(() => {
+      this.resetEmail = '';
+      this.$scope.forgotForm.$setPristine();
+      this.$scope.forgotForm.$setUntouched();
+      this.$mdToast.show(this.$mdToast.simple()
+        .textContent('Username sent; please check your email.')
```

Comment:
We might want to add a disclaimer that the email will only be sent if the username actually exists.